src/compiler/objective_c_plugin.cc

```diff
@@ -52,15 +52,16 @@ inline ::grpc::string ImportProtoHeaders(
 
   ::grpc::string base_name = header;
   grpc_generator::StripPrefix(&base_name, "google/protobuf/");
+  ::grpc::string file_name = "GPB" + base_name;
```

Comment:
@ thomasvl - could you confirm that your change follows the pattern of 1) moving WKT files from `objectivec/google/protobuf/` to `objectivec/` and 2) add `GPB` prefix to the headers?

Reply:
I've already check with Thomas, we just need to update paths from `objectivec/google/protobuf/`to `objectivec/` and add `GPB` prefix to the headers. 

He mentioned that we also need to set the min version of protobuf to the release(**v3.12.0**) where they shipped this. I've check !ProtoCompiler.podspec, the version is **3.12.2**, looks like we don't need to change it.